packages/editor-ui/src/stores/assistant.store.ts

```diff
@@ -106,6 +110,10 @@ export const useAssistantStore = defineStore(STORES.ASSISTANT, () => {
 			).length,
 	);
 
+	const isSupportChatSessionInProgress = computed(() => {
+		return currentSessionId.value !== undefined && chatSessionError.value === undefined;
```

Comment:
once we add credential helper, we will have to store the type. For now, this works as we only have two types.